### ISSUE TITLE
[Bug]: Every town falls to rebel side in quick succession after AFRF reprisals

### ISSUE THREAD

**Issue by Kerrski** [igorkis-scrts/A3-Antistasi-Plus#58]
### Describe the bug

Almost half of the towns on the map fell to rebel side after winning AFRF reprisals mission. It happened a second time and the other half of the towns fell to rebels.

![image](https://user-images.githubusercontent.com/89209697/130096386-b9fec7d8-3310-4396-aa9d-7b67bac435dc.png)

![image](https://user-images.githubusercontent.com/89209697/130096420-c73f86b9-d7d6-483e-b4ab-8552a8d09421.png)

From what I understand this shouldn't happen so quickly. We were only on war level 3 when the first reprisals mission started and we only controlled one resource just south of our base plus two extra towns. We hadn't been farming missions and were mostly just doing hit and run activities to build up weapon stocks.

### How to reproduce

Win the AFRF reprisals mission.

### Version

1.7

### Have you altered the code?

No

### Map

Takistan

### What server?

Private dedicated server

### Mods

```html
RHSUSAF, RHSAFRF, RHSGREF, RHSSAF, CBA_A3, Cup Terrains - Core, Cup Terrains - Maps, 3CB Factions
```

**Comment by Kerrski** [comment]
Just a note - we only won the reprisals mission twice because the Russians attacked Rasman right next to the USAF airbase twice.

**Comment by igorkis-scrts** [comment]
![image](https://user-images.githubusercontent.com/6746043/130116350-b6f34934-4f7e-4f3d-84ae-5ea6e41f8e7b.png)

From what i reading from Antistasi Community discord, Punishment missions are insanely broken, and your bug is already well known by community.

I'm going to cut this mission from Antistasi Plus in 1.7.4.

**Comment by igorkis-scrts** [comment]
Fixed in 1.7.4.
